casper/src/test/rholang/BasicWalletTest.rho

```diff
@@ -1,145 +1,158 @@
 //scalapackage coop.rchain.rholang.wallet
 
 //requires BasicWallet, MakeMint, TestSet, ListOps
-@"sk"!("1388803416a5869f3d4682fb3fae738278287b80d1a5a52ddf89be8eb9dac59d") |
-@"pk"!("79d16233dedf8e8e4cc25272b2b98ac096dac9239ade20dcee5987a1c1d101c7") |
-@"sig0"!("04f9a5f223fdf61ceaa290d04df2c171892d46cee31faad55061896b16340558d27e6b27d55a7133ec6ebf07433a059d1e3c2fd2ea864ad01de69e05f860180e") |
-@"sig1"!("ed1d624af32b9ec1d252c1df8faac112e1a5b9839ca4ba4027105453e8b23c71768fb80bd324c3605a8412cb51c1d86b641538e276965a5b2a742a10d136770c") |
-@"invalidSig"!("b8cea97ee7afdab7eb6ebabe8fde9ca8f9dbd3b877e8fe6b6cfd4bcef5cfdcfccec9fd8de9e39df4b6f5a2f7b6d9da8cffef48fab85faddbaab8ebca4af4ab07") |
+new
+  rl(`rho:registry:lookup`), TestSetCh,
+  test1, test2, test3, test4, test5
+in {
+  rl!(`rho:id:6wnujzcraztjfg941skrtbdkdgbko8nuaqihuhn15s66oz8ro5gwbb`, *TestSetCh) |
+  for(@(_, TestSet) <- TestSetCh) {
+    @"sk"!("1388803416a5869f3d4682fb3fae738278287b80d1a5a52ddf89be8eb9dac59d") |
```

Comment:
Here and below, sending & receiving on forgeable channels will fail soon.  I think you can just say `new sk, pk, sig0 in ...` and remove the `@"..."` around each name.

Also, though I can't click on the line because it's not in the diff, didn't the API change from `@(mint, "makePurse")!(...)` to `mint!("makePurse", ...)` ?

Reply:
Those changes are made in https://github.com/rchain/rchain/pull/1723 . This PR just focuses on `TestSet` and dependencies. I opened it before we decided to do the rest as one big PR, so I left this one open and the other PR has all the other changes. The other PR actually includes this one, so if you want to see all the changes then you can just look at that one and I can close this one.